plugins/Eq/EqControlsDialog.cpp

```diff
@@ -45,113 +45,287 @@ EqControlsDialog::EqControlsDialog( EqControls *controls ) :
 	m_controls = controls;
 	setAutoFillBackground( true );
 	QPalette pal;
-	pal.setBrush( backgroundRole(), PLUGIN_NAME::getIconPixmap( "artwork" ) );
+	pal.setBrush( backgroundRole(), PLUGIN_NAME::getIconPixmap( "EqLayout1BG" ) );
 	setPalette( pal );
-	setFixedSize( 350, 275 );
+	setFixedSize( 500, 500 );
+	QGridLayout * MLayout = new QGridLayout(this);
```

Comment:
The variable `MLayout ` should follow the same lowercase naming convention used in the rest of the file.